### PR TITLE
feat(ui): Event detail DebugMeta improvements

### PR DESCRIPTION
**Pull request by matejminar**
Changes made:

- optimized searching/filtering to be faster
- better experience on mobile devices
- changed cog icon to search icon
- fixed full page reload when navigating to DIFs in settings
- used ClipBox, because some people find annoying that debug images' inner container is scrollable and gets in the way

#sync-getsentry

### CODE REVIEW

File: src/sentry/static/sentry/app/components/clippedBox.jsx [hunk 1]
```diff
@@ -42,8 +42,21 @@ class ClippedBox extends React.Component {
...
+    const renderedHeight = ReactDOM.findDOMNode(this).offsetHeight; // eslint-disable-line react/no-find-dom-node
+    if (renderedHeight < this.props.clipHeight) {
+      this.reveal();
+    }
+  }
+
   reveal = e => {
-    e.stopPropagation();
+    if (e) {
+      e.stopPropagation();
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/clippedBox.jsx:58]: maybe you could change it to `e?.stopPropagation()` ?

File: src/sentry/static/sentry/app/components/events/interfaces/debugmeta.jsx [hunk 2]
```diff
@@ -182,7 +195,7 @@ class DebugImage extends React.PureComponent {
 
         <ImageInfoGroup>
           <Formatted>{formatAddress(startAddress, IMAGE_ADDR_LEN)}</Formatted> &ndash;{' '}
-          <br />
+          <AddressDivider />
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/interfaces/debugmeta.jsx:198]: 👏

**Review by priscilawebdev — APPROVED** [review]
Kudos!! it looks great! 🚀

File: src/sentry/static/sentry/app/components/clippedBox.jsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/clippedBox.jsx]: yes! 🔥

**Pull request merged**